app/models/competition.rb

```diff
@@ -738,16 +741,16 @@ def trainee_delegate_ids
     @trainee_delegate_ids || trainee_delegates.map(&:id).join(",")
   end
 
-  def enable_v2_registrations!
-    update_column :uses_v2_registrations, true
+  def uses_microservice_registrations?
+    self.registration_version_v2?
   end
 
-  def uses_new_registration_service?
-    self.uses_v2_registrations
+  def uses_new_registration_system?
+    self.uses_microservice_registrations? || self.registration_version_v3?
```

Comment:
`registration_version_v3` is provided automatically by the enums?

Reply:
Yes!